set.mm

```diff
@@ -180585,9 +180550,12 @@ obviously not a bijection (by Cantor's theorem ~ canth2 ), and in fact
       BVCVENQUPNVFOVGVHVIVJVK $.
   $}
 
-  $( Extend ~ gcdmultiple so ` N ` can be an integer.  (Contributed by Scott
-     Fenton, 18-Apr-2014.)  (Revised by Mario Carneiro, 19-Apr-2014.) $)
-  gcdmultiplez $p |- ( ( M e. NN /\ N e. ZZ ) -> ( M gcd ( M x. N ) ) = M ) $=
+  $( Obsolete proof of ~ gcdmultiple as of 12-Jan-2024.  Extend ~ gcdmultiple
```

Comment:
```suggestion
  $( Obsolete proof of ~ gcdmultiplez as of 12-Jan-2024.  Extend ~ gcdmultiple
```
(Also: for obsolete proofs, I generally delete the rest of the comment, since it is identical, but I guess that's how you prefer.)

Reply:
Correction done. Since the OLD theorems are deleted after a year anyway, it does not matter if the comment is kept or not. (If the OLD theorem is not near the new theorem as in this case, I think it is better to keep the old comment.)